SYCL/Matrix/joint_matrix_nvptx_double.cpp

```diff
@@ -0,0 +1,152 @@
+// REQUIRES: gpu, cuda
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -Xsycl-target-backend
```

Comment:
Can we use a different prefix  in the name like "tensorcore" rather than "nvptx"?
Also, can you add a comment that in the tensor core case, JIT compilation is not supported an specifying the arch argument is necessary?
I know this will be added as part of the doc but repetition can be useful here.

Reply:
Yeah both suggestions sound good to me.

Reply:
Made the change